### PR TITLE
 Update rke2 releases august 2021

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

**Review by cwayne18 — DISMISSED** [review]

File: channels-rke2.yaml [hunk 1]
```diff
@@ -119,7 +119,7 @@ releases:
       rancher-vsphere-csi:
         repo: rancher-charts
         version: 2.1.000
-  - version: v1.21.4-rc1+rke2r1
```
Comment (deniseschannon) [inline channels-rke2.yaml:122]: I would like this PR to also include removing the 1.21.3 release to prep for the 2.6 release.

**Comment by deniseschannon** [comment]
https://github.com/rancher/rancher/issues/33880

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: channels-rke2.yaml (hunk 1 above)
Comment (briandowns) [reply channels-rke2.yaml:122]: 1.21.3 removed

**Comment by deniseschannon** [comment]
Small nit: can you update from &charts to &charts-v1?

**Comment by briandowns** [comment]
Updated.

**Review by deniseschannon — APPROVED** [review]

**Comment by deniseschannon** [comment]
Please do not merge until given the green light as this will remove having 2 versions of RKE2 in our release.

**Pull request merged**